hspec-core/src/Test/Hspec/Core/Formatters/V2.hs

```diff
@@ -293,7 +293,7 @@ defaultFailedFormatter = do
         Error _ e -> withFailColor . indent $ (("uncaught exception: " ++) . formatException) e
 
       writeLine ""
-      writeLine ("  To rerun use: --match " ++ show (joinPath path))
+      writeLine ("  To rerun use: --match " ++ ['"'] ++ joinPath path ++ ['"'])
```

Comment:
I think this will not behave sane if  `joinPath path` contains `"`-characters.

Reply:
That's right.
I did it this way because I didn't want to add too many dependencies, but it seems that unicode-show is still needed.

Reply:
I added unicode-show.